underlay/src/main/resources/config/datamapping/sd/entity/cpt4/entity.json

```diff
@@ -26,7 +26,7 @@
       "rootIdFieldName": "id",
       "maxDepth": 10,
       "keepOrphanNodes": false,
-      "cleanHierarchyNodesWithZeroCounts": true
+      "cleanHierarchyNodesWithZeroCounts": false
```

Comment:
This should be left to `true` else we see all zero counts.  we do not want zero counts concepts